### DISCUSSION TITLE
Does Martin return Z (height) coordiante in 3d geometries?

### DISCUSSION THREAD

**Discussion by phpia** [maplibre/martin#314] [General]
Hi, does Martin return Z (height) coordiante in 3d geometries?

Or always cut 3d coordiantes to lat and lng?

Thanks!!!

**Comment by radoslavirha** [comment]
I always get tiles without Z. Did you find out how to solve it? Thanks

**Reply by phpia** [reply]
I didnt find a solution over MVT, I am using Geojson losing perfomance :)

**Comment by nyurik** [comment]
Possibly related [3D MVT discussion](https://github.com/mapbox/vector-tile-spec/issues/111) -- sadly no decision has been made yet.  Would be good to be able to support 3D vector tiles too.
